server/supernova/server/server.cpp

```diff
@@ -207,7 +210,7 @@ static void name_current_thread(int thread_index) {
 }
 
 static void set_daz_ftz(void) {
-#ifdef __SSE__
+#if defined(BOOST_ARCH_X86_64) || (BOOST_HW_SIMD_X86 >= BOOST_X86_SSE_VERSION)
```

Comment:
this can simply be `BOOST_HW_SIMD_X86 >= BOOST_X86_SSE_VERSION`, and you can remove the `architecture.h` include